DataFormats/MuonReco/src/Muon.cc

```diff
@@ -247,6 +299,45 @@ unsigned int Muon::RPClayerMask( ArbitrationType type ) const
 
 }
 
+
+unsigned int Muon::ME0layerMask( ArbitrationType type ) const
+{
+   unsigned int totMask(0);
+   unsigned int curMask(0);
+   for( std::vector<MuonChamberMatch>::const_iterator chamberMatch = muMatches_.begin();
+	 chamberMatch != muMatches_.end(); chamberMatch++ )
+   {
+      if(chamberMatch->me0Matches.empty()) continue;
+	 
+      ME0DetId rollId = chamberMatch->id.rawId();
+      const int region = rollId.region();
+
+      const int layer  = rollId.layer();
+      int me0Layer = chamberMatch->station();
+
+      //Not sure if this correctly computes the layer...
+      //chamberMatch->station() on me0s now returns "1" no matter what, since there is only one station
+      //This means as it is written now there is a bit shift equal to which number layer the me0Match is
+      //ME0Segments have detId.layer() of 1 every time, with a roll of "-1"
+      if (region==1 && region==-1) {
```

Comment:
Nevermind, the typo is actually the "&&", it should be an "||".  It is meant as a *probably* unnecessary check that the region is either 1 (plus endcap) or -1 (minus endcap).  Fixed in upcoming push.